e2e/txpool_test.go

```diff
@@ -276,3 +278,124 @@ func TestTxPool_TransactionCoalescing(t *testing.T) {
 		"To address balance mismatch after gap transaction",
 	)
 }
+
+func TestGasLimit(t *testing.T) {
```

Comment:
This should be TestPriceLimit (as below), not gaslimit?

Reply:
Fixed in https://github.com/0xPolygon/polygon-sdk/pull/151/commits/d5e92b6f828f644bfbf9e025fc62d6bea106cf42